velox/docs/monitoring/stats.rst

```diff
@@ -31,11 +31,25 @@ Memory Arbitration
        an spillable operator.
        This stats only applies for spillable operators.
    * - memoryReclaimWallNanos
-     - kNano
+     - kNanos
      - The memory reclaim execution time of an operator during the memory
        arbitration. It collects time spent on disk spilling or file write.
        This stats only applies for spillable operators.
    * - reclaimedMemoryBytes
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+Task Execution
+------------------
+.. list-table::
+   :widths: 40 10 50
+   :header-rows: 1
+
+   * - Stats Name
+     - Type
+     - Description
+   * - hashtable.buildWallNanos
```

Comment:
Let's document all hashtable.xxx stats. Perhaps, in a separate PR that this PR will depend on.

Reply:
Sure, done. https://github.com/facebookincubator/velox/pull/9255